code/modules/mining/equipment.dm

```diff
@@ -573,9 +573,8 @@
 			L.underlays -= marked_image
 			qdel(marked_image)
 			marked_image = null
-			var/backstab = check_target_facings(user, L)
 			var/def_check = L.getarmor(type = "bomb")
-			if(backstab == FACING_INIT_FACING_TARGET_TARGET_FACING_PERPENDICULAR || backstab == FACING_SAME_DIR)
+			if(get_turf(user) == get_turf(get_step(L, turn(L.dir, 180))) || get_turf(user) == get_turf(get_step(L, turn(L.dir, 135))) || get_turf(user) == get_turf(get_step(L, turn(L.dir, 225))))		//Checks the 3 tiles behind the target.
```

Comment:
augh why this is not what I told you to do

    var/backstab_dir = get_dir(user, L)
    if((user.dir & backstab_dir) && (L.dir & backstab_dir))

And you don't even really need the user dir check, you only really need the L.dir check.